tests/browser/spec/XAxisLabels.spec.js

```diff
@@ -0,0 +1,66 @@
+import chai, { expect } from "chai";
+import { mount } from "enzyme";
+import React from "react";
+import sinon from "sinon";
+import sinonChai from "sinon-chai";
+import { XAxisLabels, XYPlot } from "src/index.js";
+chai.use(sinonChai);
+
+// XAxisLabels tests must run in browser since XAxisLabels uses measureText
+
+describe("XAxisLabel", () => {
+  const width = 500;
+  const height = 300;
+
+  it("Check how many labels are created and where", () => {
+    const props = {
+      width,
+      height,
+      scaleType: { x: "linear", y: "linear" },
+      margin: { top: 11, bottom: 22, left: 33, right: 44 }
+    };
+    const chartStyle = { marginBottom: "10px" };
+    const functions = {
+      onMouseEnterLabel: sinon.spy(),
+      onMouseMoveLabel: sinon.spy(),
+      onMouseLeaveLabel: sinon.spy()
+    };
+    const tree = (
+      <div>
+        <div style={chartStyle}>
+          <XYPlot
+            width={400}
+            height={150}
+            xDomain={[-20, 20]}
+            yDomain={[-20, 20]}
+          >
+            <XAxisLabels {...functions} />
+            <XAxisLabels position="top" distance={2} tickCount={5} />
+          </XYPlot>
+        </div>
+      </div>
+    );
+    const rendered = mount(tree).find(XAxisLabels);
+    const first = rendered.first();
+    // each tick is a g and there's a wrapper around all g's, hence the 10 and 6
+    expect(first.find("g")).to.have.length(10);
+    expect(rendered.at(1).find("g")).to.have.length(6);
```

Comment:
The 10 is for the number of ticks right? What's the 6 length in reference too?

Reply:
this test sets up two sets of ticker (line 37 and 38), so this is testing that the default is 9 and the manual setting is 5

Reply:
got it 👍